configuration/cardano/shelley-dummy-genesis.json

```diff
@@ -1,21 +1,21 @@
 {
     "activeSlotsCoeff": 0.05,
-    "systemStart": "2017-09-23T21:44:51Z",
+    "systemStart": "2017-09-23T09:44:51Z",
```

Comment:
This does not seem correct. 

`db-sync` now reads the Byron and Shelley configs and compares the two dates. The 24 hour date is the correct one.